.github/workflows/CI.yml

```diff
@@ -42,6 +42,11 @@ env:
   WIN_SIGNED_PKG_BUCKET: aoc-aws-signer-signed-artifact-dest
   WIN_UNSIGNED_PKG_FOLDER: OTelCollectorAuthenticode/AuthenticodeSigner-SHA256-RSA
   WIN_SIGNED_PKG_FOLDER: OTelCollectorAuthenticode/AuthenticodeSigner-SHA256-RSA
+  INTEGRATION_TEST_FIRST_REGION: us-east-2
+  INTEGRATION_TEST_AMP_FIRST_REGION: "https://aps-workspaces.us-east-2.amazonaws.com/workspaces/ws-1de68e95-0680-42bb-8e55-67e7fd5d0861"
+  INTEGRATION_TEST_SECOND_REGION: us-west-2
+  INTEGRATION_TEST_AMP_SECOND_REGION: "https://aps-workspaces.us-west-2.amazonaws.com/workspaces/ws-cd278045-1d6e-4550-a207-c5046a1b40b8"
```

Comment:
I feel like the naming of these variables can be a little bit confusing, could we rename them as `INTEGRATION_TEST_REGION_A` and B or something that makes it more explicit that it's part of a group?

Reply:
Maybe INTEGRATION_TEST_REGION_US_WEST_2 and INTEGRATION_TEST_REGION_US_EAST_2

Reply:
```INTEGRATION_TEST_REGION_A``` sounds similar to sub-region to me and mostly I don't specify the region since if we decided to change the region, we don't need to change the naming convention.

Reply:
@khanhntd I agree that it would sound like a sub-region. Could we instead do something like `INTEGRATION_TEST_REGION_ONE` and `INTEGRATION_TEST_REGION_TWO` because it would be easier to tell that these two variables are together.

Reply:
```suggestion
  INTEG_TEST_1_REGION: us-east-2
  INTEG_TEST_1_AMP: "https://aps-workspaces.us-east-2.amazonaws.com/workspaces/ws-1de68e95-0680-42bb-8e55-67e7fd5d0861"
  INTEG_TEST_2_REGION: us-west-2
  INTEG_TEST_2_AMP: "https://aps-workspaces.us-west-2.amazonaws.com/workspaces/ws-cd278045-1d6e-4550-a207-c5046a1b40b8"
```

Reply:
Reasonable. Fixed it to ```INTEGRATION_TEST_REGION_1``` and similar variables. Nice opinions thought.